docs/1 Introduction/2 Install Starport.md

```diff
@@ -14,6 +14,8 @@ Local Starport installation requires the follow software be installed and runnin
 
 - [Node.js >=12.19.0](https://nodejs.org/)
 
+Make sure that your `$PATH` includes `$GOPATH/bin` with `export PATH=$PATH:$GOPATH/bin`.
```

Comment:
Or `$PATH` needs to include `$HOME/go/bin` if `$GOPATH` isn't set. cc @barriebyron.

Reply:
Let's then say:

```
Make sure that your `$PATH` includes `$HOME/go/bin` with `export PATH=$PATH:$HOME/go/bin`.
```

Reply:
I mean it depends, if `$GOPATH` is set, the other option probably won't work.

Reply:
Appending `$HOME/go/bin` should always work.